editor/src/core/performance/performance-regression-tests.spec.tsx

```diff
@@ -64,7 +64,7 @@ describe('React Render Count Tests -', () => {
 
     const renderCountAfter = renderResult.getNumberOfRenders()
     // if this breaks, GREAT NEWS but update the test please :)
-    expect(renderCountAfter - renderCountBefore).toMatchInlineSnapshot(`343`)
+    expect(renderCountAfter - renderCountBefore).toMatchInlineSnapshot(`344`)
```

Comment:
I wonder how accurate this is with the mocked ReactDnD we're having to use with jest 😬

Reply:
The puppeteer tests don't appear to have deviated so I think we're good